modules/miner/miner.go

```diff
@@ -194,10 +199,15 @@ func New(cs modules.ConsensusSet, tpool modules.TransactionPool, w modules.Walle
 // Close terminates all ongoing processes involving the miner, enabling garbage
 // collection.
 func (m *Miner) Close() error {
+	if err := m.tg.Stop(); err != nil {
```

Comment:
At the top of `threadedMine` is

```go
if err := m.tg.Add(); err != nil {
    return
}
```

Is this sufficient?  Or should `tg.IsStopped()` be checked explicitly in the for loop or something?